### PR TITLE
reset all filters functionality added #236

### PR DESCRIPTION
**Pull request by BearerOP**
This pr resolves the issue #236 as there is no functionality of **"Reset all filter"**  in Filters sections of View Jobs page.

https://github.com/user-attachments/assets/168edfbf-e4f4-4ecc-85a0-4730dcd5cae4

### CODE REVIEW

**Comment by aakash2330** [comment]
Please also make sure that the results reset too , it's not very ui friendly if the user have to also click on apply after resetting

**Comment by BearerOP** [comment]
> Please also make sure that the results reset too , it's not very ui friendly if the user have to also click on apply after resetting

Noted and Completed! ✅


https://github.com/user-attachments/assets/4009bae8-2c6b-4f47-be8b-8d93a00acb9b

**Comment by VineeTagarwaL-code** [comment]
nice this is good

**Review by VineeTagarwaL-code — COMMENTED** [review]
looks good checking the pr in local right now

**Comment by VineeTagarwaL-code** [comment]
the workflow fails +  doesn't updates the url 
image

**Comment by BearerOP** [comment]
I forget to commit the latest modification, I guess it's fine now. @VineeTagarwaL-code

**Comment by VineeTagarwaL-code** [comment]
fix the conflicts will merge

**Comment by BearerOP** [comment]
> fix the conflicts will merge

Done ✅

File: src/layouts/job-filters.tsx [hunk 1]
```diff
@@ -41,27 +41,39 @@ const JobFilters = ({ searchParams }: { searchParams: JobQuerySchemaType }) => {
     },
   });
 
-  const formValues = form.watch();
+  async function handleFormSubmit(data: JobQuerySchemaType) {
+    await jobFilterQuery(
```
Comment (aakash2330) [inline src/layouts/job-filters.tsx:45]: please use the new setQueryParams hook , or if you're just resetting the page you can just reload the page without query params , something like  router.replace(pathname);

**Pull request closed without merging**